### PR TITLE
Update Elasticsearch to 6.8.21

### PR DESCRIPTION
**Pull request by lbakerchef**
Also nuke ES 7 which we will never ship.

Signed-off-by: Lincoln Baker <lbaker@chef.io>
Signed-off-by: Tim Smith     <tsmith@chef.io>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by lbakerchef** [comment]
https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/3574

File: omnibus/config/software/elasticsearch.rb [hunk 1]
```diff
@@ -25,14 +25,14 @@
 
 relative_path "elasticsearch-#{version}"
 
-version "6.8.18" do
+version "6.8.20" do
```
Comment (tas50) [inline omnibus/config/software/elasticsearch.rb:28]: Since we only use one and this is in the server you might as well nuke .20

**Review by PrajaktaPurohit — APPROVED** [review]
Let's keep es7 for now. Other than that looks good to merge.

File: omnibus/config/software/elasticsearch.rb (hunk 1 above)
Comment (lbakerchef) [reply omnibus/config/software/elasticsearch.rb]: Done.

**Review by PrajaktaPurohit — APPROVED** [review]

**Pull request merged**